packages/bech32/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@penumbra-zone/bech32",
+  "version": "0.0.1",
+  "description": "Tools for manipulating Penumbra bech32 strings",
+  "type": "module",
+  "exports": {
+    ".": "./src/index.ts"
+  },
+  "publishConfig": {
+    "files": [
+      "./dist"
+    ],
+    "exports": {
+      ".*": {
+        "import": "./dist/index.js",
+        "types": "./dist/index.d.ts"
+      }
+    }
+  },
+  "scripts": {
+    "build": "vite build",
+    "clean": "rm -rfv dist",
+    "lint": "eslint \"**/*.ts*\""
+  },
+  "dependencies": {
+    "@buf/penumbra-zone_penumbra.bufbuild_es": "1.8.0-20240313211643-35db357c277d.1",
```

Comment:
This is in the global package right?

Reply:
yes, but we do refer to its types. if we publish this as a package we need the output package.json to contain a dep